owncloudDomain/src/main/java/com/owncloud/android/domain/exceptions/DeepLinkException.kt

```diff
@@ -0,0 +1,25 @@
+/**
+ * ownCloud Android client application
+ *
+ * @author Manuel Plazas Palacio
+ *
+ * Copyright (C) 2022 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.owncloud.android.domain.exceptions
+
+class DeepLinkException() : Exception(INVALID_FORMAT)
```

Comment:
`()` are not needed if the class doesn't have parameters in the constructor

Reply:
Done